### ISSUE TITLE
Count put multiple services injectable into constructor

### ISSUE THREAD

**Issue by litstrong** [litstrong/Dragon#67]
```
constructor(http : Http, http1 : Http);
```

Doesn't allow this.

**Comment by litstrong** [comment]
at line 9, file /Users/litstrong/Projects/Dragon/frontend/app/login.tsArgument of type 'typeof LoginComponent' is not assignable to parameter of type 'Type'.
Component({
  selector: 'login-selector',
  viewBindings: [],
})

**Comment by litstrong** [comment]
at line 13, file /Users/litstrong/Projects/Dragon/frontend/app/login.tsArgument of type 'typeof LoginComponent' is not assignable to parameter of type 'Type'.
View({
  templateUrl: 'app/login-template.html',
  directives: [NgFor, NgIf, NgClass, RouterLink, RouterOutlet],
})

**Comment by litstrong** [comment]
```
var injector = Injector.resolveAndCreate([TodoStore, TodoFactory, TodoApp]);
var store = injector.get(TodoStore);
```

Might use this one to implement it first.

**Comment by litstrong** [comment]
Know annotations and decorators:
http://blog.thoughtram.io/angular/2015/05/03/the-difference-between-annotations-and-decorators.html

**Comment by litstrong** [comment]
Could not find the reason, might they only want to support on parameter in the constructor.

Inject a service and make that service a multiple parameters constructor.

**Comment by litstrong** [comment]
Will mark this closed, and put issue left in the #68.
